packages/create-sitecore-jss/src/common/steps/transform.ts

```diff
@@ -14,7 +14,7 @@ import {
 import { diffLines, diffJson, Change } from 'diff';
 import { BaseArgs } from '../args/base';
 
-const COPY_ONLY_REGEX = /\.(gif|jpg|jpeg|tiff|png|svg|ashx|ico|pdf|jar)$/;
+const COPY_ONLY_REGEX = /\.(gif|jpg|jpeg|tiff|png|svg|ashx|ico|pdf|jar|html)$/;
```

Comment:
this will filter out all angular components, as they use `.html` file extension. Need to filter specifically `index.html`